src/components/useraccount/TabReturn/TabReturn.js

```diff
@@ -22,6 +22,11 @@ export default {
     updateSelectedItems(items) {
       this.selectedItems = items;
     },
+    closeModal() {
```

Comment:
@plaskevich  You can use [this abstraction](https://github.com/commercetools/sunrise-spa/blob/master/src/components/common/shared.js#L82-L98) instead. and $router.go is not needed.

Reply:
I changed the code